src/test/java/com/checkmarx/jenkins/CheckmarxScanBuilderTest.java

```diff
@@ -103,6 +110,39 @@ public void failWrongCheckmarxInstallation() throws Exception {
         this.jenkins.assertBuildStatus(Result.FAILURE, build);
         this.jenkins.assertLogContains("Please configure the build properly and retry.", build);
     }
+    @Test
+    public void successCheckmarxScanAndVerifyArtifacts() throws Exception {
```

Comment:
please update ethe test name inder when_given_then(should) concentration and have comments of the steps cause it's hard to understand what it does